tools/PI/DevHome.PI/SettingsUi/AddToolControl.xaml

```diff
@@ -92,8 +93,12 @@
             </Grid>
         </Border>
 
-        <StackPanel Orientation="Horizontal" Grid.Row="4" HorizontalAlignment="Right" Margin="0,12,0,0">
-            <Button x:Uid="AddToolsOkButton" x:Name="OkButton"  Click="OkButton_Click" Width="100"/>
+        <ToggleSwitch 
+            x:Uid="IsPinnedToggleSwitch" x:Name="IsPinnedToggleSwitch"
+            Grid.Row="4" Margin="0,12,0,0" HorizontalAlignment="Left" IsOn="True"/>
+        
+        <StackPanel Orientation="Horizontal" Grid.Row="5" HorizontalAlignment="Right" Margin="0,12,0,0">
+            <Button x:Uid="AddToolsOkButton" x:Name="OkButton" Click="OkButton_Click" Width="100"/>
             <Button x:Uid="AddToolsCancelButton" x:Name="CancelButton" Margin="6,0,0,0" Width="100"/>

```

Comment:
Suggest adding Spacing to the surrounding StackPanel instead of Margin to the Button.

Reply:
Fixed in 2nd commit.